pyproject.toml

```diff
@@ -1,7 +1,174 @@
+    "tvOS",
+    "watch",
+    "watchOS",
+    "wearos",
+    "web",
+    "pyscript",
+    "pyodide",
+]
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: BSD License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3 :: Only",
+    "Topic :: Software Development",
+    "Topic :: Utilities",
+]
+dependencies = [
+    # Dependencies required at runtime are set as ranges to ensure maximum
+    # compatibility with the end-user's development environment.
+    #
+    # Any dependency that is part of the "core python toolchain" (e.g. pip,
+    # wheel) specify a minimum version, but no maximum, because we always want
+    # the most recent version.
+    "importlib_metadata >= 4.4; python_version <= '3.9'",
+    "packaging >= 22.0",
+    "pip >= 23.1.1",
+    "setuptools >= 60",
+    "wheel >= 0.37",
+    "build >= 0.10",
+    # For the remaining packages: We set the lower version limit to the lowest
+    # possible version that satisfies our API needs. If the package uses semver,
+    # we set a limit to prevent the next major version (which will potentially
+    # cause API breakages). If the package uses calver, we don't pin the upper
+    # version, as the upper version provides no basis for determining API
+    # stability.
+    "cookiecutter >= 2.3.1, < 3.0",
+    "dmgbuild >= 1.6, < 2.0; sys_platform == 'darwin'",
+    "GitPython >= 3.1, < 4.0",
+    "platformdirs >= 2.6, < 5.0",
+    "psutil >= 5.9, < 6.0",
+    "requests >= 2.28, < 3.0",
+    "rich >= 12.6, < 14.0",
+    "tomli >= 2.0, < 3.0; python_version <= '3.10'",
+    "tomli_w >= 1.0, < 2.0",
+]
+
+[project.optional-dependencies]
+# Extras used by developers *of* briefcase are pinned to specific versions to
+# ensure environment consistency.
+dev = [
+    "coverage[toml] == 7.3.2",
+    "coverage-conditional-plugin == 0.9.0",
+    "pre-commit == 3.5.0",
+    "pytest == 7.4.3",
+    "pytest-xdist == 3.4.0",
+    "setuptools_scm == 8.0.4",
+    "tox == 4.11.3",
+]
+docs = [
+    "furo == 2023.9.10",
+    "pyenchant == 3.2.2",
+    # Sphinx 7.2 deprecated support for Python 3.8
+    "sphinx == 7.1.2 ; python_version < '3.9'",
+    "sphinx == 7.2.6 ; python_version >= '3.9'",
+    "sphinx_tabs == 3.4.4",
+    "sphinx-autobuild == 2021.3.14",
+    "sphinx-copybutton == 0.5.2",
+    "sphinxcontrib-spelling == 8.0.0",
+]
+
+[project.urls]
+Homepage = "http://beeware.org/briefcase"
+Funding = "https://beeware.org/contributing/membership/"
+Documentation = "http://briefcase.readthedocs.io/en/latest/"
+Tracker = "https://github.com/beeware/briefcase/issues"
+Source = "https://github.com/beeware/briefcase"
```

Comment:
Warning-zealous PyCharm is reminding me the `http://` scheme is insecure...

Reply:
Well that definitely needs to be updated...

Reply:
I swear... it doesn't matter how many times you audit for this, you find new ones. A whole bunch of http references found throughout the project.

Reply:
Gotta get this IDE that nanny's you all day ;)